prov/verbs/src/fi_verbs.c

```diff
@@ -80,7 +145,7 @@ int fi_ibv_create_ep(const char *node, const char *service,
 		goto out;
 
 	if (!node && !rai_hints.ai_dst_addr) {
-		if (!rai_hints.ai_src_addr && !service) {
+		if (!rai_hints.ai_src_addr) {
```

Comment:
This change prevents listening on wildcard address when the app specifies just the port but no source address info.